src/main/java/com/sngular/kloadgen/extractor/extractors/JsonExtractor.java

```diff
@@ -175,7 +192,11 @@ private List<FieldValueMapping> extractArrayInternalFields(
             FieldValueMapping.builder()
                              .fieldName(name)
                              .fieldType(value.getType() + "-array" + ((StringUtils.isNotEmpty(breadCrumb) && breadCrumb.endsWith("[]")) ? "-array" :
-                                 (StringUtils.isNotEmpty(breadCrumb) && breadCrumb.endsWith("[:]")) ? "-map" : breadCrumb))
```

Comment:
Check if the formatter its ok, the concatenation of this one should be in one or two lines instead of this